services/reviewService/components/Title.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+const Title = () => <h2 className="reviews-section-title">What people are saying</h2>;
```

Comment:
It looks like this doesn't need its own file per se. This probably doesn't pertain to you, considering it's on the frontend and was most likely inherited.

Reply:
I agree. I will probably end up refactoring the frontend even more when I have down time.